packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -18,13 +19,20 @@
   @SuppressWarnings("unchecked")
   FlutterWebView(Context context, BinaryMessenger messenger, int id, Map<String, Object> params) {
     webView = new WebView(context);
+
+    methodChannel = new MethodChannel(messenger, "plugins.flutter.io/webview_" + id);
+    methodChannel.setMethodCallHandler(this);
+
+    applySettings((Map<String, Object>) params.get("settings"));
+
+    if (params.containsKey("javascriptChannelNames")) {
```

Comment:
nit, optional: Consider breaking out the "javascriptChannelNames" magic string into a private constant on this class or a named variable. It'll prevent typo problems with getting the value.

Reply:
done